api/jobs.go

```diff
@@ -1514,3 +1516,31 @@ type JobEvaluateRequest struct {
 type EvalOptions struct {
 	ForceReschedule bool
 }
+
+// nomad action is used to run a pre-defined command inside a running task.
```

Comment:
This is "Go style" for the docstring of public API:

```suggestion
// ActionExec is used to run a pre-defined command inside a running task.
```